drivers/soundwire/intel.c

```diff
@@ -1670,18 +1670,11 @@ int intel_master_process_wakeen_event(struct platform_device *pdev)
 	intel_shim_wake(sdw, false);
 
 	/*
-	 * wake up master and slave so that slave can notify master
-	 * the wakeen event and let codec driver check codec status
+	 * wake up master and slave will be re-enumerated, then resumed by
+	 * sdw_handle_slave_alerts(). So that slave can notify master the
+	 * wakeen event and let codec driver check codec status
```

Comment:
looks good but the comment can be improved a bit, e.g.

resume the master, which will generate a bus reset and result in Slaves re-attaching and be re-enumerated. The SoundWire physical device which generated the wake will trigger an interrupt, which will in turn cause the corresponding Linux Slave device to be be resumed and the Slave codec driver to check the status.

Reply:
@plbossart comment updated.